_data/sidebars/home_sidebar.yml

```diff
@@ -1,278 +1,188 @@
 # This is your sidebar TOC. The sidebar code loops through sections here and provides the appropriate formatting.
 
-entries:
-- title: Sidebar
-  product: Menu
-  levels: one
-  folders:
   - title: Get Started
-    output: web
     folderitems:
     - title: Welcome to Portworx Docs
-      url: /index.html
-      output: web
+      url: /
     - title: Get Started ASAP
-      url: /get-started-asap.html
-      output: web
+      url: /getting-started/asap.html
     - title: Get Started with PX-Developer
-      url: /get-started-px-developer.html
-      output: web
+      url: /getting-started/px-developer.html
     - title: Get Started with PX-Enterprise
-      url: /get-started-px-enterprise.html
-      output: web
+      url: /getting-started/px-enterprise.html
   - title: Install and Run
-    output: web
     folderitems:
     - title: Overview
-      url: /overview.html
-      output: web
+      url: /install/
     - title: Run with Docker
-      url: /run-with-docker.html
-      output: web
+      url: /install/docker.html
     - title: Run with CoreOS
-      url: /run-with-docker-coreos.html
-      output: web
+      url: /install/coreos.html
     - title: Run with Docker Compose
-      url: /run-with-compose.html
-      output: web
+      url: /install/docker-compose.html
     - title: Run as a Docker Plugin 
-      url: /run-as-docker-pluginv2.html
-      output: web   
+      url: /install/docker-plugin.html
   - title: Manage Volumes
-    output: web
     folderitems:
     - title: Create and Manage Storage Volumes
-      url: /create-manage-storage-volumes.html
-      output: web
+      url: /manage/create-and-manage.html
     - title: Shared Volumes
-      url: /shared-volumes.html
-      output: web
+      url: /manage/shared-volumes.html
     - title: Volume Sets
-      url: /volume-sets.html
-      output: web
+      url: /manage/volume-sets.html
     - title: Class of Service
-      url: /cos.html
-      output: web
+      url: /manage/class-of-service.html
     - title: Snapshots
-      url: /snapshot.html
-      output: web
+      url: /manage/snapshots.html
     - title: Volume Update
-      url: /volume-update.html
-      output: web
+      url: /manage/volume-update.html
     - title: Encrypted Volumes
-      url: /encrypted-volumes.html
-      output: web     
+      url: /manage/encrypted-volumes.html
   - title: Schedulers
-    output: web
     folderitems:
     - title: Run Portworx with Mesosphere DCOS
```

Comment:
then have url: /scheduler/mesosphere-dcos.html be

url: /scheduler/mesosphere-dcos/install.html